prisma/schema.prisma

```diff
@@ -68,19 +75,21 @@ model User {
   accessStatus  AccessStatus @default(PENDING) @map("access_status")
   accessMode    AccessMode   @default(FULL_ACCESS) @map("access_mode")
 
-  class Class? @relation(fields: [classId], references: [id], onDelete: SetNull)
+  class       Class?        @relation(fields: [classId], references: [id], onDelete: SetNull)
+  information Information[]
```

Comment:
1-1 relation https://www.prisma.io/docs/concepts/components/prisma-schema/relations/one-to-one-relations

Reply:
updated